src/utils/constants/events.js

```diff
@@ -91,20 +99,13 @@ export const pastEventData = [
 ];
 
 export const futureEventData = [
-  {
-    imageBackgroundColor: null,
-    date: 'June 2019',
-    imageLink: 'https://res.cloudinary.com/drumsensei/image/upload/v1560255997/cea-name-side_xzh2ti.png',
-    meetupLink: 'https://www.meetup.com/ReactJSDallas/events/pbbdwnyzjbpb/',
-    speaker1: 'Mike Mathew - Test your components',
-    venue: "Call-Em-All",
-  },
   {
     imageBackgroundColor: null,
     date: 'July 2019',
     imageLink: 'https://res.cloudinary.com/drumsensei/image/upload/v1560256211/intuit-logo_cupxcj.jpg',
     meetupLink: 'https://www.meetup.com/ReactJSDallas/events/pbbdwnyzkbmb/',
-    speaker1: 'TBD',
+    speaker1: 'Travis Baker - Playing Hooky with React: Effectively Use Primitive, Custom, & 3rd Party React Hooks',
+    speaker1: 'Jaime Liz - Generators: How Do They Work & Why Should I Care',
```

Comment:
one of these supposed to be `speaker2`? :)

Reply:
DOH! And _that_ is why we shouldn't self-merge. 😞